app/classes/api2/image_api.rb

```diff
@@ -60,8 +60,13 @@ def query_params
         with_votes: parse(:boolean, :has_votes),
         quality: parse_range(:quality, :quality),
         confidence: parse_range(:confidence, :confidence),
-        ok_for_export: parse(:boolean, :ok_for_export)
-      }.merge(parse_names_parameters)
+        ok_for_export: parse(:boolean, :ok_for_export),
+        observation_query: parse_observation_query_parameters.compact
```

Comment:
Is there a reason you're calling `.compact` twice?

Reply:
good catch, fixed